qiskit_alice_bob_provider/local/translation_plugin.py

```diff
@@ -150,7 +164,8 @@ def pass_manager(
             basis_gates=pass_manager_config.basis_gates,
             method='translator',
             approximation_degree=pass_manager_config.approximation_degree,
-            coupling_map=pass_manager_config.coupling_map,
+            # coupling_map=pass_manager_config.coupling_map,
```

Comment:
I guess this should be removed.

Reply:
Actually, this was used during a test and it was important to restore it! Thanks for pointing this out